earthpy/clip.py

```diff
@@ -146,14 +217,14 @@ def clip_shp(shp, clip_obj):
     if not any(shp.intersects(clip_obj.unary_union)):
         raise ValueError("Shape and crop extent do not overlap.")
 
-    # Multipolys / point / line don't clip properly
-    if "Multi" in str(clip_obj.geom_type) or "Multi" in str(shp.geom_type):
-        raise ValueError(
-            "Clip doesn't currently support multipart geometries. Consider "
-            "using .explode to create unique features in your GeoDataFrame"
-        )
-
-    if shp["geometry"].iloc[0].type == "Point":
+    # if there are multi points
```

Comment:
I think we should delete this comment, because it's redundant with the line below
```suggestion
```